src/modules/0.0.22/aws_codedeploy/sql/after_install.sql

```diff
@@ -0,0 +1,140 @@
+ -- prevents update on revision table
```

Comment:
interesting, I think historically we leave this up to the mapper so on apply it puts the record back based on what is in the cloud. I'm curious, why did you decide to do it this way?

Reply:
i have no way to identify a revision... there is no single id, arn, name... i can use. So i thought it would be better to prevent at SQL , because i do not have a trusted way to do it from our mappers.